### DISCUSSION TITLE
Package name

### DISCUSSION THREAD

**Discussion by sgvignali** [CREWdecisions/elicitr#1] [Ideas]
I made some research for the package name and it looks like we can use `elicitr`

This is the output of `pak::pkg_name_check("elicitr")`:

![image](https://github.com/user-attachments/assets/fcbcd1f9-b318-4b1e-99f9-474a0ed8e512)

Do we all agree on this name?

> [!WARNING] 
> Although there is a stereo amplifier named [Elicit-R](https://www.rega.co.uk/download/rega-elicit-r-english-manual-mk2-april-2014.pdf) from the British **Rega** company.
> Another possibility would be to use `elctr` instead of `elicitr`, something similar to `dplyr` instead of, I guess,  `d-plier`.

**Comment by maude-v** [comment]
I think elctr sounds like something about electricity.. 
I had other ideas otherwise:
- eliciter (It is a french word though, so I don't know if it's clever)
- elicitize 
- expertise

What do you think?
